app/uk/gov/hmrc/agentmapping/controller/MappingController.scala

```diff
@@ -166,7 +184,65 @@ class MappingController @Inject()(
       }
   }
 
+  def createPreSubscriptionMapping(utr: Utr) = Action.async { implicit request =>
+    implicit val hc = fromHeadersAndSession(request.headers, None)
+    authorised(AuthProviders(GovernmentGateway) and AffinityGroup.Agent)
+      .retrieve(Retrievals.credentials and Retrievals.agentCode and Retrievals.allEnrolments) {
+        case credentials ~ agentCodeOpt ~ allEnrolments =>
+          captureIdentifiersAndAgentCodeFrom(allEnrolments, agentCodeOpt) match {
+            case Some(identifiers) =>
+              identifiers
+                .map(identifier => createMappingInRepository(utr, identifier, credentials.providerId))
+                .reduce((f1, f2) => f1.flatMap(b1 => f2.map(b2 => b1 & b2)))
+                .map(
+                  allConflicted =>
+                    if (allConflicted)
+                      Conflict
+                    else
+                    Created)
+            case None => Future.successful(Forbidden)
```

Comment:
403 response when they have no eligibile enrolments is not mentioned in the README.md